apps/playwright/tests/applicant/application.spec.ts

```diff
@@ -73,12 +86,163 @@ test.describe("Application", () => {
+    await expect(
+      application.page.getByText(
+        /you don’t have any career timeline experiences yet./i,
+      ),
+    ).toBeVisible();
+    await application.saveAndContinue();
+    // Expect error when no experiences added
+    await expect(application.page.getByRole("alert")).toContainText(
+      /please add at least one experience/i,
+    );
+
+    // Add an experience
+    await application.page
+      .getByRole("link", { name: /add a new experience/i })
+      .click();
+    await application.addExperience();
+    await expect(application.page.getByRole("alert")).toContainText(
+      /successfully added experience!/i,
+    );
+    await expect(
+      application.page.getByText("1 education and certificate experience"),
+    ).toBeVisible();
+    await application.saveAndContinue();
+    await application.waitForGraphqlResponse("Application");
 
-    pool = createdPool;
+    // Education experience page - step four
+    await expectOnStep(application.page, 4);
+    await application.saveAndContinue();
+    await expect(
+      application.page.getByText(/this field is required/i),
+    ).toBeVisible();
+    await application.page
+      .getByRole("radio", { name: /i meet the 2-year post-secondary option/i })
+      .click();
+    await application.page
+      .getByRole("checkbox", { name: /qa testing at playwright university/i })
+      .click();
+    await application.saveAndContinue();
+
+    // Skills requirement page - step five
+    await expectOnStep(application.page, 5);
+    await application.page
+      .getByRole("link", { name: /let's get to it/i })
+      .click();
+    await expect(
+      application.page.getByText(
+        /this required skill must have at least 1 career timeline experience associated with it/i,
+      ),
+    ).toHaveCount(2);
+    await application.saveAndContinue();
+    await expect(
+      application.page.getByText(
+        /please connect at least one career timeline experience to each required technical skill/i,
+      ),
+    ).toBeVisible();
+
+    // Connect same experience to two different skills.
+    await application.page
+      .getByRole("button", {
+        name: new RegExp(
+          `connect a career timeline experience to ${skillOne.name.en}`,
+          "i",
+        ),
+      })
+      .click();
+    await application.connectExperience("QA Testing at Playwright University");
+
+    await application.page
+      .getByRole("button", {
+        name: new RegExp(
+          `connect a career timeline experience to ${skillTwo.name.en}`,
+          "i",
+        ),
+      })
+      .click();
+    await application.connectExperience("QA Testing at Playwright University");
+
+    await expect(
+      application.page.getByText(
+        /please connect at least one career timeline experience to each required technical skill and ensure each skill has details about how you used it/i,
+      ),
+    ).toBeHidden();
+    await application.saveAndContinue();
```

Comment:
This is the :moneybag: maker right here, and it looks good to me :+1: nice work. 
I was checking it out manually, I wonder if we should check if adding an experience to optional technical skills, won't remove it from other skills?

Reply:
@yonikid15 What would be the purpose of that check? The original [bug](https://github.com/GCTC-NTGC/gc-digital-talent/issues/8727) and its [fix](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8726) were not related to whether the skill was optional or not. If you think this would an important test (I don't, but 🤷), perhaps a new issue would make more sense since it would be out of the scope of this work?

Reply:
Oh if the bug was only related to the technical skills then this looks good!